static/app/views/automations/components/automationForm.tsx

```diff
@@ -0,0 +1,105 @@
+import {useEffect} from 'react';
+import styled from '@emotion/styled';
+import {flattie} from 'flattie';
+
+import {Flex} from 'sentry/components/container/flex';
+import {Button} from 'sentry/components/core/button';
+import SelectField from 'sentry/components/forms/fields/selectField';
+import Form from 'sentry/components/forms/form';
+import FormModel from 'sentry/components/forms/model';
+import {useDocumentTitle} from 'sentry/components/sentryDocumentTitle';
+import {DebugForm} from 'sentry/components/workflowEngine/form/debug';
+import {IconAdd, IconEdit} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import AutomationBuilder from 'sentry/views/automations/components/automationBuilder';
+import {
+  AutomationBuilderContext,
+  initialState,
+  useAutomationBuilderReducer,
+} from 'sentry/views/automations/components/automationBuilderContext';
+import ConnectedMonitorsList from 'sentry/views/automations/components/connectedMonitorsList';
+
+const FREQUENCY_OPTIONS = [
+  {value: '5', label: t('5 minutes')},
+  {value: '10', label: t('10 minutes')},
+  {value: '30', label: t('30 minutes')},
+  {value: '60', label: t('60 minutes')},
+  {value: '180', label: t('3 hours')},
+  {value: '720', label: t('12 hours')},
+  {value: '1440', label: t('24 hours')},
+  {value: '10080', label: t('1 week')},
+  {value: '43200', label: t('30 days')},
+];
+
+const model = new FormModel({initialData: {...flattie(initialState), frequency: '1440'}});
```

Comment:
the form should be put into state so that it isn't reused

```tsx
const [model] = useState(() => new FormModel({initialData: {...flattie(initialState), frequency: '1440'}}));
```

Reply:
fixed in https://github.com/getsentry/sentry/pull/89233/commits/c3b26dc656daeca63eb34f33b18ab0b48f40ce34